pkg/scheduler/algorithmprovider/registry.go

```diff
@@ -164,7 +164,7 @@ func applyFeatureGates(config *schedulerapi.Plugins) {
 	if !utilfeature.DefaultFeatureGate.Enabled(features.DefaultPodTopologySpread) {
 		// When feature is enabled, the default spreading is done by
 		// PodTopologySpread plugin, which is enabled by default.
-		klog.Infof("Registering SelectorSpread plugin")
+		klog.InfoS("Registering SelectorSpread plugin")
```

Comment:
Any difference here?

Reply:
Yeah,new  Structured Logging migration uses InfoS.

Reply:
but for this case, I cannot see any benefit from the change, `Infof` is still there.

Reply:
Yes,for this case,may not see the difference.But when there are variables, the format is different.
According to the following rules, Infof will be replaced by InfoS.
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md
https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging